00-open-a-shapefile.Rmd

```diff
@@ -6,15 +6,17 @@ authors: [Joseph Stachelek, Leah Wasser]
 contributors: [Megan A. Jones, Sarah Newman]
 dateCreated:  2015-10-23
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
-packagesLibraries: [raster, rgdal]
+packagesLibraries: [rgdal, raster]
 category: 
 tags: [module-1]
 mainTag: GIS-Spatial-Data
-description: "This lesson explains the how to open and plot point, line, and polygon shapefiles in R."
+description: "This lesson explains the how to open and plot point, line and
+polygon shapefiles in `R`."
```

Comment:
Megan - the `r` doesn't render in the description properly. Please remove this from the YAML throughout. This only populates the description on the lesson list pages.